modules/base/rendering/pointcloud/renderableinterpolatedpoints.cpp

```diff
@@ -324,9 +324,9 @@ void RenderableInterpolatedPoints::initializeShadersAndGlExtras() {
             std::filesystem::path path = absPath("${MODULE_BASE}/shaders/pointcloud/");
```

Comment:
```suggestion
            std::filesystem::path path = absPath("${MODULE_BASE}/shaders/pointcloud");
```

The `/` syntax below automatically adds the `/` already